src/taipy/templates/scenario-management/{{cookiecutter.root_folder_name}}/config/config_with_toml.py

```diff
@@ -0,0 +1,5 @@
+from taipy import Config
+
```

Comment:
The `clean_data()` method in `config/config.py` should also be here.
```suggestion


def clean_data(df, replacement_type):
    df = df.fillna(replacement_type)
    return df

```

Reply:
Actually, it should not be in config.py nor config_with_toml.py. It should be in algos/algos.py

Reply:
Yes you're right, config.py should only contain configuration methods.